utils/virtcontainers-setup.sh

```diff
@@ -59,7 +59,7 @@ cp -r ${busybox_bundle}/* ${pause_bundle}/
 echo -e "Move to ${tmpdir}/${virtcontainers_build_dir}"
 pushd ${tmpdir}/${virtcontainers_build_dir}
 echo "Clone cni"
-git clone https://github.com/containernetworking/cni.git
+git clone https://github.com/containernetworking/plugins.git
```

Comment:
We only vendor this package because of the libcni we need inside virtcontainers. But what we need from this script are the plugins. And since the plugins have been moved to a different repo, there are no reason we want to follow a specific version, does that make sense ?